platform/config/environments/staging.json

```diff
@@ -14,7 +14,7 @@
 		"platform": {
 			"scheme": "https",
 			"host": "amp-dev-staging.appspot.com",
-			"port": ""
+			"port": "8080"
```

Comment:
This will cause Grow to build URLs with port like `amp-dev-staging.appspot.com:8080/index.html` if I'm not mistaken. Should we add another host object for Grow?

Reply:
good find, this isn't needed anymore.

Reply:
fixed